site/docs/3_manage_nifi/1_manage_clusters/1_deploy_cluster/1_quick_start.md

```diff
@@ -28,7 +28,10 @@ volumeBindingMode: WaitForFirstConsumer
 Remember to set your NiFiCluster CR properly to use the newly created StorageClass.
 :::
 
-As a pre-requisite NiFi requires Zookeeper so you need to first have a Zookeeper cluster if you don't already have one.
+To manage its cluster and states, NiFi needs Zookeeper or rights on the Kubernetes cluster to manage `Leases` and `ConfigMaps` resources in the namespace where it is deployed.
+
+In the case of Zookeeper, you must first have a Zookeeper cluster if you don't already have one.
+Otherwise, you need to provide the corresponding role to the NiFi cluster's `ServiceAccount`.
```

Comment:
I feel this page needs reworked a bit to make it clear what the options are. There are 3 special cases we discuss:
 - installing zookeeper in vanilla kubernetes
 - configuring a service account for Kubernetes state management
 - special configuration needed for installing zookeeper in openshift

Can we consolidate it into (massively abbreviated):

## State Management
As of NiFi 2.0, NiFi supports two state management mechanisms: Kubernetes and Zookeeper. If you're deploying NiFi 1.x, then you must use Zookeeper. Kubernetes becomes an option from NiFi 2.0.

### Installing Zookeeper
... zookeeper installation instructions ...

#### Openshift Specific Configuration
... special zookeeper openshift instructions ...

### Enabling Kubernetes State Management
... explain that you must create the ServiceAccount/Role/RoleBinding or use the nifi-cluster helm chart, which creates this for you...